### PR TITLE
Update navigation

### LINKED ISSUE #48: Update navigation
Right now we have two pages:
Dashboard
Chorals

Perhaps we could add these pages:
**Home** 
- Their current chorals
- Their account information
- Stats about their chorals, like how many requests/sec they are using
- Their api keys, secrets (if any)

**Events** #20
**Dashboard** #44
**Chorals** #33 (Add devices to sidebar) and #34 (List children)

### PR DESCRIPTION
**Pull request by james-woo**
fixes #48 

Added some new pages: Home, Events, Devices
Now you can also view devices

### CODE REVIEW

File: views/sidebar.pug [hunk 1]
```diff
@@ -15,47 +9,21 @@ mixin menu(googleUser)
...
+      a.w3-bar-item.w3-button.w3-padding(href='/chorals')
         i.fa.fa-share-alt.fa-fw
         |   Chorals
       a.w3-bar-item.w3-button.w3-padding(href='/logout')
         i.fa.fa-sign-out.fa-fw
         |   Log Out
       br
-      br
-  // Overlay effect when opening sidebar on small screens
-  #myOverlay.w3-overlay.w3-hide-large.w3-animate-opacity(onclick='w3_close()', style='cursor:pointer', title='close side menu')
```
Comment (andreirtaylor) [inline views/sidebar.pug]: Please put this back. The menu so much nicer when we can open and close it.

File: views/sidebar.pug [hunk 2]
```diff
@@ -15,47 +9,21 @@ mixin menu(googleUser)
...
-          i.fa.fa-user
-        a.w3-bar-item.w3-button(href='#')
-          i.fa.fa-cog
     hr
-    .w3-container
-      h5 Dashboard
     .w3-bar-block
-      a.w3-bar-item.w3-button.w3-padding-16.w3-hide-large.w3-dark-grey.w3-hover-black(href='#', onclick='w3_close()', title='close menu')
-        i.fa.fa-remove.fa-fw
-        |   Close Menu
```
Comment (andreirtaylor) [inline views/sidebar.pug]: See below

File: views/sidebar.pug (hunk 1 above)
Comment (james-woo) [reply views/sidebar.pug]: Ok

File: views/home.pug [hunk 3]
```diff
@@ -0,0 +1,13 @@
+extends layout
+
+block content
+  header.w3-container(style='padding-top:22px')
+    
+
+  // Footer
+  footer.w3-container.w3-padding-16.w3-light-grey
```
Comment (andreirtaylor) [inline views/home.pug]: Remove this and add it to the layout

File: views/layout.pug [hunk 4]
```diff
@@ -47,12 +53,3 @@ body.w3-light-grey
       if flash.error
         div.flash.bg-danger= flash.error
       block content
-
-      // Footer
-      footer.w3-container.w3-padding-16.w3-light-grey
```
Comment (andreirtaylor) [inline views/layout.pug]: Add this back in but move it one tab to the right. Then its on every page.

File: views/chorals.pug [hunk 5]
```diff
@@ -28,5 +23,5 @@ block content
           td= choral.sampleRate
           td= choral.func
           td
-            button.btn.btn-danger.delete-choral-btn(data-choral-id=choral.choralId)
```
Comment (andreirtaylor) [inline views/chorals.pug]: Not your issue but can you set choral.func to  td= viewHelpers.truncateWithDots(choral.func, 30)

**Review by andreirtaylor — CHANGES_REQUESTED** [review]

File: models/choral.js [hunk 6]
```diff
@@ -68,13 +68,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
   });
 };
 
-choralSchema.statics.findAllForUser = function (user, cb) {
+choralSchema.statics.findChoralsForUser = function (user, cb) {
   this.find({ userId: user._id }, (err, chorals) => {
```
Comment (andreirtaylor) [inline models/choral.js]: If we are going to have devices as well as chorals we should only return chorals on this page. We also will still probably need a find all for user when adding children.

File: models/choral.js [hunk 7]
```diff
@@ -68,13 +68,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
...
     if (err) return cb(err, null);
     cb(null, chorals);
   });
 };
 
+choralSchema.statics.findDevicesForUser = function (user, cb) {
+  this.find({ $and: [ { userId: user._id }, { choralType: "device" } ] }, (err, devices) => {
+    if (err) return cb(err, null);
```
Comment (andreirtaylor) [inline models/choral.js:80]: I think we can just do 
`this.find({ $and: [ { userId: user._id }, { choralType: "device" } ] }, cb)`

**Review by andreirtaylor — APPROVED** [review]

**Comment by andreirtaylor** [comment]
Rebase and we should be good to go.Just needs a rebase.

**Comment by andreirtaylor** [comment]
This looks great!!!!!

**Pull request merged**